src/BloomBrowserUI/pageChooser/js/page-chooser.ts

```diff
@@ -111,8 +135,8 @@ class PageChooser {
             var dataBookArray = $( "div[data-book='bookTitle']", pageData );
             var collectionTitle = $( dataBookArray.first() ).text();
             // Add title and container to dialog
-            var collectionToAdd = $( collectionHTML).clone();
-            $( collectionToAdd ).find( ".collectionCaption" ).text(collectionTitle);
+            var collectionToAdd = $(collectionHTML).clone();
+            this.setLocalizedText($(collectionToAdd).find(".collectionCaption"), 'Template.', collectionTitle);
```

Comment:
When choosing the id, you're doing a bit of UI design. Where in that localization tree would you want this to show up? Well, we have a root for EditTab, so it can at least go under there. We also have an element for AddPageDialog (which itself should move to under EditTab), but this might not be the only place we refer to page templates. But... probably it is. So I'd put it there. So perhaps this would work well: 'AddPageDialog.TemplateBooks.'.